app/services/IdentityService.scala

```diff
@@ -88,8 +90,14 @@ object PersonalDataJsonSerialiser {
         "billingAddress3" -> personalData.address.town,
         "billingAddress4" -> personalData.address.countyOrState,
         "billingPostcode" -> personalData.address.postCode,
-        "billingCountry" -> personalData.country.name
-      ),
+        "billingCountry"  -> personalData.country.name
+      ).++(
+         telephoneNumber.map {t =>
+        Json.obj(
```

Comment:
The indentation makes this particularly hard to read. Also, `fold` might be more idiomatic here